### PR TITLE
Update datamodel for timeline chart and sentiment chart

### PR DESCRIPTION
**Pull request by kshll6**
Data model for Sentiment chart and Timeline Chart is updated, such that it is less complex

### CODE REVIEW

File: alvenir/guidance/v1/guidance.proto [hunk 1]
```diff
@@ -1,67 +1,73 @@
...
+message SentimentData{
+    string speaker = 1;
+    repeated SentimentCoordinates data_points = 2;
 }
 
-message FloatDataConcat {
-  string title = 1;
-  FloatData caller_data = 2;
-  FloatData advisor_data = 3;
+message SentimentConcat {
```
Comment (sorenmulli) [inline alvenir/guidance/v1/guidance.proto:61]: What is a concat? :)

File: alvenir/guidance/v1/guidance.proto [hunk 2]
```diff
@@ -1,67 +1,73 @@
...
 
 // for timeline chart
-message StringData {
-  repeated string x = 1;
-  repeated string y = 2;
-  string speaker = 3;
+message TimeInterval {
+    string start = 1;
+    string end = 2;
+}
```
Comment (sorenmulli) [inline alvenir/guidance/v1/guidance.proto]: A lot of string values seem to describe numeric data - why are they strings?

File: alvenir/guidance/v1/guidance.proto [hunk 3]
```diff
@@ -1,67 +1,73 @@
...
+    float x = 1;
+    float y = 2;
+}
 
-message StringDataConcat {
-  string title = 1;
-  StringData caller_data = 2;
-  StringData advisor_data = 3;
+message SentimentData{
+    string speaker = 1;
```
Comment (sorenmulli) [inline alvenir/guidance/v1/guidance.proto]: Could be an enum but maybe not necessary

File: alvenir/guidance/v1/guidance.proto [hunk 4]
```diff
@@ -1,67 +1,73 @@
 syntax = "proto3";
+
 package alvenir.guidance.v1;
 
 
 message LlmPromptAnswer {
-  string prompt = 1;
-  string context = 2;
-  string answer = 3;
+    string prompt = 1;
```
Comment (sorenmulli) [inline alvenir/guidance/v1/guidance.proto]: Something with indentation where this repo is inconsistent. It seems like someone suggests 2 space indent [1] but I also see in many of the other files that it is mixed 

[1] https://protobuf.dev/programming-guides/style/#standard-file-formatting

**Review by sorenmulli — CHANGES_REQUESTED** [review]
Kool DATA

File: alvenir/guidance/v1/guidance.proto (hunk 1 above)
Comment (kshll6) [reply alvenir/guidance/v1/guidance.proto:61]: concatenation of advisor and caller data :upside_down_face:

File: alvenir/guidance/v1/guidance.proto (hunk 2 above)
Comment (kshll6) [reply alvenir/guidance/v1/guidance.proto]: Fordi formatet i python bliver lavet om til HH:MM::ss:ms så dumme javascript forstår det

Comment (sorenmulli) [reply alvenir/guidance/v1/guidance.proto]: Også n_words og n_fill_words? :)

File: alvenir/guidance/v1/guidance.proto (hunk 3 above)
Comment (kshll6) [reply alvenir/guidance/v1/guidance.proto]: Will add todo for later, when other fixes needed

File: alvenir/guidance/v1/guidance.proto (hunk 4 above)
Comment (kshll6) [reply alvenir/guidance/v1/guidance.proto]: fixed

File: alvenir/guidance/v1/guidance.proto (hunk 2 above)
Comment (kshll6) [reply alvenir/guidance/v1/guidance.proto]: samme argument as above

**Review by sorenmulli — APPROVED** [review]
*L*ets *G*o *T*o (the) *M*oon

**Comment by kshll6** [comment]
> _L_ets _G_o _T_o (the) _M_oon

TODAY!

**Pull request merged**